net.crepererum.breitbandmessung.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.crepererum.breitbandmessung</id>
```

Comment:
this doesn't match the download URL / license url that is on `breitbandmessung.de`

Reply:
So I use the upstream ID everywhere, even though they don't officially endorse it? Because the PR checklist states:

> I own the domain used in the application ID ...

Reply:
This line is a little bit misleading. It means that Upstream should own to ID; not the Maintainer

Reply:
This should be `de.breitbandmessung.Breitbandmessung`.

Reply:
Shall I re-create the PR than? (because the branch name contains the ID)

Reply:
the branch name is completely irrelevant.

Reply:
done